source/guide_ethercalc.rst

```diff
@@ -0,0 +1,163 @@
+.. highlight:: console
+
+.. author:: Achim | pxlfrk <hallo@pxlfrk.de>
+
+.. tag:: spreadsheet
+.. tag:: lang-node-js
+.. tag:: web
+.. tag:: collaborative-editing
+
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/ethercalc.png
+      :align: center
+
+#########
+Ethercalc
+#########
+
+.. tag_list::
+
+EtherCalc_ is a web spreadsheet based on node-js and redis. People can edit the same document at the same time. Everybody's changes are instantly reflected on all screens. Work together on inventories, survey forms, list management, brainstorming sessions and more!
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`domains <web-domains>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`web-backends <web-backends>`
+  * :lab:`redis <guide_redis>`
+  
+
+License
+=======
+
+Redis is distributed under different licenses, including `Common Public Attribution License (Socialtext Inc.)`_, `Apache License 2.0 (SheetJS)`_ and `MIT License`_.
+
+Prerequisites
+=============
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 12:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '12'
+ [isabell@stardust ~]$
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+We’ll also need :lab:`Redis <guide_redis>`, so follow the Redis guide and come back when it’s running.
+
+
+Installation
+============
+
+Install node module
+-------------------
+
+Install the node module using the provided node packet manager `npm`: 
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ npm install ethercalc
```

Comment:
This drops `node_modules` directory into the users home. Please put it into a directory to keep things clean.
```suggestion
 [isabell@stardust ~]$ mkdir ethercalc
 [isabell@stardust ~]$ cd ethercalc
 [isabell@stardust ethercalc]$ npm install ethercalc
```

Reply:
fixed in the guide.